bitnami/moodle/templates/_helpers.tpl

```diff
@@ -68,7 +68,11 @@ Return  the proper Storage Class
 Moodle&trade; credential secret name
 */}}
 {{- define "moodle.secretName" -}}
-{{- coalesce .Values.existingSecret (include "common.names.fullname" .) -}}
+{{- if .Values.existingSecret }}
+    {{- printf "%s" .Values.existingSecret -}}
+{{- else -}}
+    {{- printf "%s" (include "common.names.fullname" .) -}}
+{{- end -}}
```

Comment:
I don't see right now any reason to change this piece of code. `coalesce` do exactly [the same](https://helm.sh/docs/chart_template_guide/function_list/#coalesce). Please amend me if I am wrong

Reply:
your right my bad
thank you for your review.